### PR TITLE
Test/int/holder

### PR DESCRIPTION
**Pull request by cjhowland**

### CODE REVIEW

File: int/tests/conftest.py [hunk 1]
```diff
@@ -253,6 +275,7 @@ async def _make_did():
 
 @pytest.fixture(scope="session")
 async def accepted_taa(backchannel):
+    backchannel.timeout = 15
     result = (await fetch_taa.asyncio(client=backchannel)).result
```
Comment (dbluhm) [inline int/tests/conftest.py:279]: This is a recent discovery of mine but I think we should take advantage of the `Client.with_timeout(new_timeout)` method:

```suggestion
    result = (await fetch_taa.asyncio(client=backchannel.with_timeout(15))).result
```

File: int/tests/conftest.py [hunk 2]
```diff
@@ -267,7 +290,7 @@ async def accepted_taa(backchannel):
 @pytest.fixture(scope="session")
 async def endorser_did(make_did, backchannel, accepted_taa):
     """Endorser DID factory fixture"""
-
+    backchannel.timeout = 15
```
Comment (dbluhm) [inline int/tests/conftest.py:293]: Same as my previous suggestion

File: int/tests/conftest.py [hunk 3]
```diff
@@ -277,12 +300,13 @@ async def endorser_did(make_did, backchannel, accepted_taa):
...
             "verkey": did.verkey,
         },
-        timeout=30,
+        timeout=15,
     )
     if response.is_error:
         raise Exception("Failed to publish DID:", response.text)
 
     LOGGER.info("DID Published")
+    backchannel.timeout = 15
```
Comment (dbluhm) [inline int/tests/conftest.py:309]: Same as previous suggestion

File: int/tests/test_connections.py [hunk 4]
```diff
@@ -1,200 +1,183 @@
...
+            client=backchannel,
+            json_body=ReceiveInvitationRequest.from_dict(lhs_conn.invitation.to_dict()),
+            auto_accept="true",
+        )
+
+        print(await get_connections.asyncio(client=backchannel))
+        message = await wait_for_message(
+            msg_type="https://github.com/hyperledger/aries-toolbox/tree/master/docs/admin-connections/0.1/connected"
+        )
+        print(message)
```
Comment (dbluhm) [inline int/tests/test_connections.py:36]: We should probably comment out or remove the `print` statements

**Review by dbluhm — COMMENTED** [review]
A few minor nitpicks but otherwise it's about ready to go!

File: int/docker-compose.yml [hunk 5]
```diff
@@ -35,6 +35,7 @@ services:
 
   tests:
     container_name: juggernaut
+    command: -k test_holder_credential_exchange
```
Comment (dbluhm) [inline int/docker-compose.yml:38]: Let's drop this line so it runs everything by default!

**Review by dbluhm — COMMENTED** [review]
One last nitpick, sorry!

File: int/docker-compose.yml (hunk 5 above)
Comment (cjhowland) [reply int/docker-compose.yml:38]: Thanks for catching that!

**Review by dbluhm — APPROVED** [review]

**Pull request merged**